core/src/main/scala/com/microsoft/azure/synapse/ml/causal/DoubleMLParams.scala

```diff
@@ -36,8 +36,40 @@ trait HasOutcomeCol extends Params {
   def setOutcomeCol(value: String): this.type = set(outcomeCol, value)
 }
 
+trait HasIteOutputCol extends Params {
+  val iteOutputCol: Param[String] = new Param[String](this,
+    "iteOutputCol",
+    "individual treatment effect output column")
+
+  def getIteOutputCol: String = $(iteOutputCol)
+
+  /**
+    * Set name of the output column for individual treatment effect
+    *
+    * @group setParam
+    */
+  def setIteOutputCol(value: String): this.type = set(iteOutputCol, value)
+
+}
+
+trait HasIteStddevOutputCol extends Params {
+  val iteStddevOutputCol: Param[String] = new Param[String](this,
+    "iteStddevOutputCol",
+    "individual treatment effect's standard deviation output column")
+
+  def getIteStddevOutputCol: String = $(iteStddevOutputCol)
+
+  /**
+    * Set name of the output column for ite standard deviation
+    *
+    * @group setParam
+    */
+  def setIteStddevOutputCol(value: String): this.type = set(iteStddevOutputCol, value)
+}
+
 trait DoubleMLParams extends Params
-  with HasTreatmentCol with HasOutcomeCol with HasFeaturesCol
+  with HasOutputCol with HasTreatmentCol with HasOutcomeCol with HasFeaturesCol
```

Comment:
Are we using "HasOutputCol" trait in the estimator or the model class? Now that you've defined `iteOutputCol`, do we still need "HasOutputCol"?

Reply:
Also, please consider combining HasIteOutputCol and HasIteStddevOutputCol into a single trait "HasITEOutput".

Reply:
will remove HasOutputCol